src/OrbitBase/FileTest.cpp

```diff
@@ -267,4 +269,52 @@ TEST(File, ReadStructureAtOffset) {
   EXPECT_STREQ(test_struct.char_array.data(), kCharArray);
 }
 
+TEST(File, MoveFile) {
+  auto tmp_file_or_error = TemporaryFile::Create();
+  ASSERT_THAT(tmp_file_or_error, HasNoError());
+  auto tmp_file = std::move(tmp_file_or_error.value());
+
+  tmp_file_or_error = TemporaryFile::Create();
+  ASSERT_THAT(tmp_file_or_error, HasNoError());
+  auto new_file = std::move(tmp_file_or_error.value());
+  new_file.CloseAndRemove();
+  std::filesystem::path new_path = new_file.file_path();
+
+  auto exists_or_error = FileExists(new_path);
+  ASSERT_THAT(exists_or_error, HasNoError());
+  ASSERT_FALSE(exists_or_error.value());
+
+  exists_or_error = FileExists(tmp_file.file_path());
+  ASSERT_THAT(exists_or_error, HasNoError());
+  EXPECT_TRUE(exists_or_error.value());
+
+  auto move_result = MoveFile(tmp_file.file_path(), new_path);
+  ASSERT_THAT(move_result, HasNoError());
+
+  exists_or_error = FileExists(new_path);
+  ASSERT_THAT(exists_or_error, HasNoError());
+  EXPECT_TRUE(exists_or_error.value());
+
+  exists_or_error = FileExists(tmp_file.file_path());
+  ASSERT_THAT(exists_or_error, HasNoError());
+  EXPECT_FALSE(exists_or_error.value());
+
+  remove(new_path.string().c_str());
+}
+
+TEST(File, ListFilesInDirectory) {
```

Comment:
Some vertical space in this test wouldn't hurt.

Reply:
done.